src/main/kotlin/org/jitsi/nlj/PacketInfo.kt

```diff
@@ -90,6 +90,11 @@ open class PacketInfo @JvmOverloads constructor(
      */
     var shouldDiscard: Boolean = false
 
+    /**
+     * Whether this packet was generated to be a probing packet.
+     */
+    var isProbing: Boolean = false
```

Comment:
i am a bit worried about `PacketInfo` 1) becoming a dumping ground for fields and, more fundamentally, holding data that doesn't really apply to all packets (true, marking an audio packet or sctp packet as `isProbing = false` is _technically_ accurate, but it still feels a bit wrong).  i don't think this change along ruins things terribly, but i worry about the precedent.  i'm probably being a bit too paranoid and if we don't think it's too bad it's something we could wait to address down the line should it become a problem.

Reply:
Would making a `VideoPacketInfo` subclass be helpful?

Reply:
I thought about that, but wondered what sort of guarantees we could make as to when one would actually be used.  For example, would we convert it on the ingress as well when we discovered a packet was video?  Or would it only be in certain places?  Also, if we were going to do this we'd have to test and cast it, but at that point would it be that much different than just testing and casting the underlying packet?  If not, then maybe we could just do different packet types to denote probing?